.github/workflows/check-codebase.yml

```diff
@@ -141,12 +141,15 @@ jobs:
           node-version: 16
           cache: yarn
 
+      - name: Downgrade global NPM version
+        run: npm install --global npm@8.4.1 ## https://github.com/blockprotocol/blockprotocol/pull/281
+
       - run: yarn install
 
       - name: Run local registry
         run: |
-          docker pull verdaccio/verdaccio
-          docker run -it --detach --rm --name verdaccio -p 4873:4873  verdaccio/verdaccio
+          docker pull verdaccio/verdaccio:5
+          docker run -it --detach --rm --name verdaccio -p 4873:4873  verdaccio/verdaccio:5
```

Comment:
Using `:5` instead of `:latest` does not change anything at the moment. Both tags point to the same image.

It seems that [v6](https://www.npmjs.com/package/verdaccio) will be released soon, so let’s get ready for that just in case.